packages/core/src/parser/cstToAstTransformer.ts

```diff
@@ -398,7 +398,7 @@ export function generateCstToAstTransfromer(parser: Parser): ICstVisitor<never,
                 }
             }
             if (ctx.FaultTolerantToken) {
-                return this.generateFieldAccessExpressionForOnlyDot(baseExpression, ctx.FaultTolerantToken);
+                return this.generateFieldAccessExpressionForOnlyDot(baseExpression, ctx.FaultTolerantToken[0]);
```

Comment:
>  When entering a single dot, the completion sometimes is not triggered

How does this fix anything?

Reply:
Pretty standard type error
It propagates to the completion range computatation, which thus fails.
This causes the completion then to not trigger.